<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralOverview</class>
 <widget class="QDialog" name="GeneralOverview">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>611</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>945</width>
    <height>550</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>General Overview</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Nick</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>General</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Kingdom</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Gender</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MaxHP</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Package</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="generalPhoto">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../image/generals/card/caocao.jpg</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="skillLabel">
          <property name="text">
           <string>Skill</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="skillTextEdit">
          <property name="maximumSize">
           <size>
            <width>211</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame1">
          <property name="minimumSize">
           <size>
            <width>241</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>271</width>
            <height>91</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <widget class="QLineEdit" name="cvLineEdit">
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="cvLabel">
             <property name="text">
              <string>CV</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="designerLineEdit">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="designerLabel">
             <property name="text">
              <string>Designer</string>
             </property>
            </widget>
           </item>
	   <item row="2" column="0">
            <widget class="QLabel" name="illustratorLabel">
             <property name="text">
              <string>Illustrator</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="illustratorLineEdit">
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="maximumSize">
           <size>
            <width>271</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>241</width>
             <height>506</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
